### PR TITLE
Packages step UI (readonly)

### PR DESCRIPTION
**Pull request by nabsul**
#505

### CODE REVIEW

**Comment by kellychoffman** [comment]
Marked as in progress while I style.

**Comment by kellychoffman** [comment]
Okay, @DanReyLop I've styled this up and have 3 requests :) 

1. Can the dimensions have a label? `Package dimension` should suffice.
2. Can the Package Contents table have a container around the whole thing? I wasn't sure how to add that. 
3. Lets add the confirmation button to this step :) I know it wasn't in the design, but when styling this I went through the UX about 20+ times. And when I get to this step after the address steps, I miss having a button that confirms the change and automatically collapses the card and brings you to the next step.

**Comment by DanReyLop** [comment]
@kellychoffman Done. Dor the most part, you can assume that everything inside the `JSX` syntax behaves exactly like HTML (except when it doesn't) :smile:

**Comment by kellychoffman** [comment]
Thanks @DanReyLop! Finished styling based on your changes and its up for review again. Loving the UX flow so far!

**Comment by allendav** [comment]
Let's _not_ include the post ID in the item title. Instead, I suggest we only use the title and, if the merchant has set one, the SKU for the product (available from Edit Product, Inventory, SKU), e.g. instead of

screen shot 2016-09-07 at 2 30 38 pm

do this:

screen shot 2016-09-07 at 2 31 35 pm

OK to punt this to a new issue

**Comment by allendav** [comment]
We're missing a space between the weight and the units:

screen shot 2016-09-07 at 2 34 10 pm

**Comment by allendav** [comment]
The weight in the expander header doesn't account for the combined weight of all packages:

screen shot 2016-09-07 at 2 54 48 pm

**Comment by DanReyLop** [comment]
@allendav If the merchant has an SKU for the product, it will appear in place of the post ID. This is how the product names are usually formatted in the Admin Dashboard, so I think it's better to leave it as-is.

About the combined weight... yeah, don't expect it to work. Putting the combined weight there has no value to the merchant (why should he care if all the packages sum 30lbs?). We need to figure out what to put there if there are multiple packages, and that's out of this sprint's scope. Unless @kellychoffman (or you) can come up with something quick.

**Comment by allendav** [comment]
> @allendav If the merchant has an SKU for the product, it will appear in place of the post ID. This is how the product names are usually formatted in the Admin Dashboard, so I think it's better to leave it as-is.

Agreed then.

> About the combined weight... yeah, don't expect it to work. Putting the combined weight there has no value to the merchant (why should he care if all the packages sum 30lbs?). We need to figure out what to put there if there are multiple packages, and that's out of this sprint's scope. Unless @kellychoffman (or you) can come up with something quick.

Hmmmm. Agreed. Let's _not_ display the weight when there are multiple packages but display the number of packages instead (e.g. "2 packages")

File: client/shipping-label/views/steps/packages/list.js [hunk 1]
```diff
@@ -1,44 +1,65 @@
...
 
 const OrderPackages = ( { packages, updateWeight, dimensionUnit, weightUnit, errors } ) => {
+	const renderItemInfo = ( item, itemIndex ) => {
+		return (
+			<div key={ itemIndex } className="wcc-package-item">
+				<div className="item-name">
+					<span className="item-name__title">
+						<a href={ item.url } target="_blank">{ item.name }</a>
+					</span>
+					{ item.attributes && <p>{ item.attributes }</p> }
```
Comment (allendav) [inline client/shipping-label/views/steps/packages/list.js]: Neat syntax

File: client/shipping-label/views/steps/packages/list.js [hunk 2]
```diff
@@ -1,44 +1,65 @@
...
 
 const renderPackageDimensions = ( pckg, dimensionUnit ) => {
 	return `${pckg.length} ${dimensionUnit} x ${pckg.width} ${dimensionUnit} x ${pckg.height} ${dimensionUnit}`;
 };
 
 const OrderPackages = ( { packages, updateWeight, dimensionUnit, weightUnit, errors } ) => {
+	const renderItemInfo = ( item, itemIndex ) => {
+		return (
+			<div key={ itemIndex } className="wcc-package-item">
+				<div className="item-name">
```
Comment (allendav) [inline client/shipping-label/views/steps/packages/list.js]: I think this className is dangerously generic. Let's do something like wcc-package-item__name or something

File: client/shipping-label/views/steps/packages/list.js [hunk 3]
```diff
@@ -1,44 +1,65 @@
...
+					<span className="item-name__title">
```
Comment (allendav) [inline client/shipping-label/views/steps/packages/list.js]: Not as dangerous as the classname above, but let's do something like wcc-package-item__title

File: client/shipping-label/views/steps/packages/list.js [hunk 4]
```diff
@@ -1,44 +1,65 @@
...
+						<a href={ item.url } target="_blank">{ item.name }</a>
+					</span>
+					{ item.attributes && <p>{ item.attributes }</p> }
+				</div>
+				<div className="item-quantity">{ item.quantity }</div>
```
Comment (allendav) [inline client/shipping-label/views/steps/packages/list.js]: Same comment as above

File: client/shipping-label/views/steps/packages/list.js [hunk 5]
```diff
@@ -1,44 +1,65 @@
...
-
-				<ul>
+			<div key={ pckgIndex }>
+				<div className="wcc-package-package-number">
+					{ sprintf( __( 'Package %d (of %d)' ), pckgIndex + 1, packages.length ) }
+				</div>
+
+				<div>
+					<div className="wcc-package-items-header">
+						<FormLegend className="item-name">{ __( 'Package contents' ) }</FormLegend>
```
Comment (allendav) [inline client/shipping-label/views/steps/packages/list.js]: Same comment as above for this line and the next

**Comment by allendav** [comment]
@DanReyLop - a couple class names need fixing - otherwise good to go

**Comment by allendav** [comment]
:shipit:

**Pull request merged**